pywinauto/unittests/test_mouse.py

```diff
@@ -128,6 +129,37 @@ def test_wheel_click(self):
         self.assertTrue("Mouse Release" in data)
         self.assertTrue("MiddleButton" in data)
 
+    # TODO: make the feature and the tests cross-platform (duration param)
+    if sys.platform == "win32":
+        def test_mouse_can_move_cursor1(self):
+            coord = (20, 10)
+            mouse.move(coord)
+            self.assertEqual(coord, win32api.GetCursorPos())
+
+        def test_mouse_can_move_cursor2(self):
```

Comment:
Given we don't reset mouse coordinates back in setUp/tearDown, maybe it's worth having one test_mouse_can_move_cursor method with several mouse.move calls and assertions, instead of separate methods? Would save a bit of the already lengthy run of unit tests.